third_party/move/move-vm/runtime/src/interpreter.rs

```diff
@@ -1606,6 +1685,18 @@ impl Frame {
             | Bytecode::VecPopBack(_)
             | Bytecode::VecUnpack(_, _)
             | Bytecode::VecSwap(_) => (),
+
+            // Since v7
```

Comment:
Do we  need this v7 distinguishing here though? We did not specify for things like `Bytecode::CastU256` which are v5 only iirc